src/replication_manager.py

```diff
@@ -68,6 +68,9 @@ def update_replication_type(self, db_state, ha_replics):
         needed = _get_needed_replication_type(self._config, self._db, db_state, ha_replics)
         logging.info('Needed replication type is %s.', needed)
 
+        if needed != current[0]:
+            logging.info('ACTION. We should change replication from {} to {}'.format(current[0], needed))
```

Comment:
I think it is not an `ACTION`, since it is not immediately preceding the described switch.